sparkpost/django/message.py

```diff
@@ -0,0 +1,53 @@
+from django.core.mail import EmailMultiAlternatives
+
+from .exceptions import UnsupportedContent
+
+
+class SparkPostMessage(dict):
+    """
+    Takes a Django EmailMessage and formats it for use with the SparkPost API.
+
+    The dictionary returned would be formatted like this:
+
+    {
+        'recipients': ['recipient@example.com'],
+        'from_email': 'from@example.com',
+        'text': 'Hello world',
+        'html': '<p>Hello world</p>',
+        'subject': 'Hello from the SparkPost Django email backend'
+    }
+    """
+
+    def __init__(self, message):
+        formatted = {
+            'recipients': message.to,
+            'from_email': message.from_email,
+            'subject': message.subject,
+            'text': message.body
+        }
+
+        if message.cc:
+            formatted['cc'] = message.cc
+
+        if message.bcc:
+            formatted['bcc'] = message.bcc
+
+        if hasattr(message, 'reply_to') and message.reply_to:
+            formatted['reply_to'] = message.reply_to
```

Comment:
Django's EmailMessage wants a list for the reply_to field but we expect a string. a naive solution might be to concat the list with semicolons:

```python
formatted['reply_to'] = ';'.join(message.reply_to)
```

Without this, using reply_to causes an API error:

```
SparkPostAPIException: Call to https://api.sparkpost.com/api/v1/transmissions returned 422, errors:

        invalid data format/type: field 'content.reply_to' is of type 'json_array', but needs to be of type 'string'
```